src/main/js/helpers/shapeSVG.js

```diff
@@ -15,3 +15,24 @@ export const getSVGObject = (shape, { posX, posY, scale }) => {
     d3Shape.options.scale = d3Shape.options.scale * scale;
     return d3Shape;
 };
+
+/**
+ * Returns an object to create a svg circle for event data point
+ *
+ * @private
```

Comment:
nit: This is not a private method. We are importing in other file and using

Reply:
[fixed](https://github.com/cerner/carbon-graphs/pull/219/commits/33b5cca47796ca87966a1c265591cd6cc253cb5a#diff-804709ab7db5d3fcf4b59f59f943c28dR20)